### PR TITLE
Fixes #1943

### PR DESCRIPTION
**Pull request by knrt10**
Signed-off-by: knrt10 <tripathi.kautilya@gmail.com>

**What this PR does / why we need it**:
This PR solves golint errors

**Which issue this PR fixes** : fixes [#1943](https://github.com/openebs/openebs/issues/1943)

### CODE REVIEW

**Comment by knrt10** [comment]
@sonasingh46 would you please review this PR. I had to close previous one.

File: pkg/task/v1alpha1/runtask_command.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ import (
 	msg "github.com/openebs/maya/pkg/msg/v1alpha1"
 )
 
+// SkipExecutionMessage will skip run command execution
```
Comment (sonasingh46) [inline pkg/task/v1alpha1/runtask_command.go]: Can you put this comment inside `const` block , above the var declaration.

**Comment by knrt10** [comment]
@sonasingh46 Updated as asked

**Comment by egurnov** [comment]
This fixed openebs/openebs#2105

**Pull request merged**